jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -26,6 +26,7 @@
 
 INFO_MSG = ("The file contains more viewable extensions. Add the '[*]' suffix"
             " to the file name to load all of them.")
+wcs_only_key = "_WCS_ONLY"
```

Comment:
Can we use the app attribute instead of re-defining it here?

Reply:
Done.